src/shogun/kernel/string/WeightedDegreePositionStringKernel.cpp

```diff
@@ -1423,7 +1423,7 @@ float64_t* CWeightedDegreePositionStringKernel::compute_scoring(
 	info.R_k = NULL;
 
 	// === main loop
-	auto pb = progress(range(num_feat * max_degree), *this->io);
+	auto pb = SG_PROGRESS(range(num_feat * max_degree));
 	for( k = 0; k < max_degree; ++k )
```

Comment:
I wonder whether having only an outer progress would be better?
@geektoni ?

Reply:
I agree. Stacked progress bars are not handled very well at the moment.